packages/input-output-logger/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = (opts) => {
+  const defaults = {
+    logger: console.log
```

Comment:
Should we *JSONStringify* by default?

Something like:

```javascript
const defaults = {
  logger: (data) => console.log(JSON.stringify(data, null, 2))
}
```

What do you think?

Reply:
The goal here is that the output would be more readable

Reply:
I'll check the output in Cloudwatch but it seems to be a good idea.